app/helpers/vm_infra_helper/textual_summary.rb

```diff
@@ -1,4 +1,5 @@
 module VmCloudHelper::TextualSummary
```

Comment:
@bdunne looks like that this helper is not in use due to the typo, initially intention was to show infra specific data on Infra VM summary screen in Virtual Machines explorer and show common VM data in Workloads explorer as we show both types on VMs (Cloud & Infra) in that explorer. It looks like https://github.com/ManageIQ/manageiq/blob/master/app/helpers/vm_helper/textual_summary.rb has cloud & Infra specific methods that solves the purpose, should be safe to remove the unused file.